src/test/java/io/cdap/plugin/salesforce/etl/SalesforceBatchSinkETLTest.java

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.salesforce.etl;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import io.cdap.cdap.api.data.format.StructuredRecord;
+import io.cdap.cdap.api.data.schema.Schema;
+import io.cdap.cdap.test.ApplicationManager;
+import io.cdap.plugin.salesforce.plugin.sink.batch.SalesforceSinkConfig;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * {@inheritDoc}
+ */
+public class SalesforceBatchSinkETLTest extends BaseSalesforceBatchSinkETLTest {
+  @Test
+  public void testInsertAccount() throws Exception {
+    String sObject = "Account";
+    Schema schema = Schema.recordOf("output",
+                                    Schema.Field.of("Name", Schema.of(Schema.Type.STRING)),
+                                    Schema.Field.of("NumberOfEmployees", Schema.of(Schema.Type.INT)),
+                                    Schema.Field.of("ShippingLatitude", Schema.of(Schema.Type.DOUBLE)),
+                                    Schema.Field.of("ShippingLongitude", Schema.of(Schema.Type.DOUBLE))
+    );
+
+    List<StructuredRecord> inputRecords = ImmutableList.of(
+      StructuredRecord.builder(schema).set("Name", "testInsertAccount1").set("NumberOfEmployees", 6)
```

Comment:
I think it will be more readable if you wrap chain method calls

Reply:
done